Composer/ScriptHandler.php

```diff
@@ -428,8 +434,8 @@ protected static function getPhpArguments()
     /**
      * Returns a relative path to the directory that contains the `console` command.
      *
-     * @param Event $event      The command event.
-     * @param string       $actionName The name of the action
+     * @param Event  $event      The command event.
+     * @param string $actionName The name of the action
```

Comment:
I just updated upstream and changes are not merged yet:

https://github.com/sensiolabs/SensioDistributionBundle/blob/master/Composer/ScriptHandler.php